web/src/settings_panel_menu.js

```diff
@@ -93,10 +93,7 @@ export class SettingsPanelMenu {
         const section = this.current_tab;
         const user_settings_tab = this.current_user_settings_tab;
 
-        if (two_column_mode()) {
-            // In one column mode want to show the settings list, not the first settings section.
-            this.activate_section_or_default(section, user_settings_tab);
-        }
+        this.activate_section_or_default(section, user_settings_tab, false);
```

Comment:
Do you want false, or some function of `two_column_mode()`?

Reply:
Yes, we need to send as per value of `two_column_mode` to retain the behavior of showing the right section if user narrows the screen after opening the overlay in wide screen.